deployment/web/pom.xml

```diff
@@ -53,13 +53,13 @@
         <dependency>
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
-            <version>3.1.0</version>
+            <version>4.0.1</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>za.co.absa.commons</groupId>
             <artifactId>commons_${scala.compat.version}</artifactId>
-            <version>0.0.27</version>
+            <version>1.3.2</version>
         </dependency>
```

Comment:
Please revert changes in the `pom.xml`. They are not related to the Front-end app and actually brake the server.